components/Snackbar/src/private/MDCSnackbarOverlayView.m

```diff
@@ -28,12 +28,12 @@
 NSString *const MDCSnackbarOverlayIdentifier = @"MDCSnackbar";
 
 // The time it takes to show or hide the Snackbar.
-NSTimeInterval const MDCSnackbarEnterTransitionDuration = 0.15f;
-NSTimeInterval const MDCSnackbarExitTransitionDuration = 0.075f;
-NSTimeInterval const MDCSnackbarLegacyTransitionDuration = 0.5f;
+NSTimeInterval const MDCSnackbarEnterTransitionDuration = (CGFloat)0.15;
```

Comment:
do we need (CGFloat) casting here and below if NSTimeInterval is a double?

Reply:
Done throughout.